keps/sig-apps/3017-pod-healthy-policy-for-pdb/README.md

```diff
@@ -267,21 +230,13 @@ Integration tests covering:
   - Correct behavior for both policies in the eviction API.
   - Feature gate disablement.
 
-
-<!--
-This question should be filled when targeting a release.
-For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
-For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
-https://storage.googleapis.com/k8s-triage/index.html
-
-- <test>: <link to test coverage>
--->
-
 [TestEvictionWithUnhealthyPodEvictionPolicy](https://github.com/kubernetes/kubernetes/blob/c8010537913422cc221cdd784936ff99817f621c/test/integration/evictions/evictions_test.go#L417): https://storage.googleapis.com/k8s-triage/index.html?test=UnhealthyPodEvictionPolicy
 
 ##### e2e tests
 
-Verify passing existing E2E and conformance tests for PDBs and Eviction.
+Introduce tests covering:
```

Comment:
Not 100% sure if having just the integration tests is not enough. But having kubelet changing the pods status should give more comprehensive testing.